tests/dbt/test_docs.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from sqlmesh.dbt.project import Project
+
+pytestmark = pytest.mark.dbt
+
+
+def test_docs_inline(sushi_test_project: Project):
```

Comment:
```suggestion
@pytest.mark.xdist_group("dbt_manifest")
def test_docs_inline():
    project_path = Path("tests/fixtures/dbt/sushi_test")
    profile = Profile.load(DbtContext(project_path))

    helper = ManifestHelper(
        project_path,
        project_path,
        "sushi",
        profile.target,
        model_defaults=ModelDefaultsConfig(start="2020-01-01"),
    )
    # Inline description in yaml
    assert helper.models()["waiters"].description == "waiters docs block"
    
    assert helper.models()["top_waiters"].description == "description of top waiters"
```

I feel something like this should fix the failures ^